### ISSUE TITLE
Bogaudio ASX crashing Rack when using three ADDR and three expanders

### ISSUE THREAD

**Issue by pgatt** [bogaudio/BogaudioModules#206]
Hi Paul Gatt here from VCV Support.

User is experiencing a crash when using VCV Rack Pro v2.1.2 Standalone on Windows 10

They added voxglitch autobreak and ghosts, and then added three instances of bogaudio addr, then three instances of the expander. When click-dragging the expander to move it next to addr rack stuttered and force closed.

Crash log: 
```
[1385.617 info src/app/Browser.cpp:90 chooseModel] Creating module Bogaudio ASX
[1385.617 info src/app/Browser.cpp:94 chooseModel] Creating module widget Bogaudio ASX
[1396.500 info src/patch.cpp:254 saveAutosave] Saving autosave C:/Users/Keith/Documents/Rack2/autosave/patch.json
[1396.503 info src/settings.cpp:470 save] Saving settings C:/Users/Keith/Documents/Rack2/settings.json
[1396.683 info src/app/ModuleWidget.cpp:828 cloneAction] Creating module Bogaudio ASX
[1396.683 info src/app/ModuleWidget.cpp:845 cloneAction] Creating module widget Bogaudio ASX
[1397.133 info src/app/ModuleWidget.cpp:828 cloneAction] Creating module Bogaudio ASX
[1397.133 info src/app/ModuleWidget.cpp:845 cloneAction] Creating module widget Bogaudio ASX
[1403.720 fatal adapters/standalone.cpp:48 fatalSignalHandler] Fatal signal 22. Stack trace:
14:  0x0
13: raise 0x7ffe9029abe0
12: abort 0x7ffe9029f1e0
11: wassert 0x7ffe9029d210
10: ZN8bogaudio8AddrSeqX13processAlwaysERKN4rack6engine6Module11ProcessArgsE 0x7ffe3c9d6430
9: ZN8bogaudio8BGModule7processERKN4rack6engine6Module11ProcessArgsE 0x7ffe3ca17460
8: ZN8bogaudio14ExpanderModuleINS_22AddrSeqExpanderMessageENS_16ExpandableModuleIS1_NS_8BGModuleEEEE7processERKN4rack6engine6Module11ProcessArgsE 0x7ffe3cae3340
7: ZN4rack6engine6Module9doProcessERKNS1_11ProcessArgsE 0x7ffe3d1a5c10
6: ZN4rack6engine12EngineWorker3runEv 0x7ffe3d1a0740
5: atomic_flag_test_and_set_explicit 0x7ffe4bf511d0
4: pthread_create_wrapper 0x7ffe7f924c20
3: beginthreadex 0x7ffe902aae30
2: endthreadex 0x7ffe902aaf80
1: BaseThreadInitThunk 0x7ffe8f407020
0: RtlUserThreadStart 0x7ffe91202630
```

Let me know if you need more information.

**Comment by mdemanett** [comment]
Confirmed, reproduced, fixed.  Will request an update on the library soon.

**Comment by mdemanett** [comment]
PS Thanks for the detailed report!

**Comment by pgatt** [comment]
Thanks Matt, my pleasure. Glad to see the fix, and thanks for being so responsive. We appreciate it!

**Comment by mdemanett** [comment]
This was fixed in [v2.1.41](https://github.com/bogaudio/BogaudioModules/releases/tag/v2.1.41).
